src/CMake/cpackWin.cmake

```diff
@@ -29,7 +29,7 @@ SET(CPACK_PACKAGE_FILE_NAME "${CPACK_PACKAGE_NAME}_${XRT_VERSION_RELEASE}.${CPAC
 message("-- ${CMAKE_BUILD_TYPE} ${PACKAGE_KIND} package")
```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`